app/controllers/application_controller/compare.rb

```diff
@@ -1223,7 +1223,7 @@ def drift_add_diff_image(idx, val)
 
   def drift_add_image_col(idx, img_src, img_bkg, val)
     html = ViewHelper.content_tag(:div, :class => img_bkg) do
-      ViewHelper.tag(:i, :class => img_src, :title => val)
+      ViewHelper.content_tag(:i, nil, :class => img_src, :title => val)
```

Comment:
@skateman does it make sense to add a test here?

Reply:
@ohadlevy well, not really. The code is ancient and IMHO it shouldn't even exist. Also the issue is not happening always and it's browser-dependent.